app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args: Any) -> Any:
+        storage_keys = storage.keys()
+        if args not in storage_keys:
```

Comment:
The check `if args not in storage_keys:` is unnecessary. You can directly check `if args not in storage:`. This simplifies the code and avoids creating an extra variable `storage_keys`.